core/src/main/java/greencity/service/impl/AdviceServiceImpl.java

```diff
@@ -75,22 +75,24 @@ public AdviceDto getAdviceByName(String language, String name) {
      * {@inheritDoc}
      */
     @Override
-    public Advice save(AdvicePostDto advicePostDTO) {
-        return adviceRepo.save(modelMapper.map(advicePostDTO, Advice.class));
+    public AdviceDto save(AdvicePostDto advicePostDTO) {
+        Advice savedAdvice = adviceRepo.save(modelMapper.map(advicePostDTO, Advice.class));
```

Comment:
Do you plan to move `*ServiceImpl` and `*ServiceImplTest` into `service` module in a separate pull request?

Reply:
The same question for `AdviceTranslateMapper`.

Reply:
Yes, don't want to mix it up.